tests/spark/test_text_operations.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import ast
+import json
+from textwrap import dedent
+
+import pytest
+# Import Operations to test
+from juicer.spark.text_operation import TokenizerOperation
+
+from juicer.spark.etl_operation import RandomSplit, Sort, Distinct, \
+    SampleOrPartition, AddRows, Intersection, Difference, Join, Drop, \
+    Transformation, Select, Aggregation, Filter, CleanMissing, AddColumns
+from tests import compare_ast, format_code_comparison
+
+
+def debug_ast(code, expected_code):
+    print
+    print code
+    print '*' * 20
+    print expected_code
+    print '*' * 20
+
+
+# Test TokenizerOperation
+
+def test_tokenizeroperation_type_simple():
```

Comment:
Separe name tokenizer_operation and add the suffix with expected result, such as _success or _failure.